source/js/main.js

```diff
@@ -231,6 +232,42 @@ let main = {
     if (document.querySelector(`#news`)) {
       ReactDOM.render(<News env={env}/>, document.querySelector(`#news`));
     }
+
+    // Science Fellowship
```

Comment:
Can we put this into a new file? `main.js` is getting pretty large and this is fairly specific to just one section.